app/graphql/cache.ts

```diff
@@ -171,20 +202,16 @@ export const createCache = () =>
         fields: {
           displayBalance: {
             read: (_, { readField, cache }) => {
-              // FIXME as RealtimePriceQuery is a singleton,
-              // could we have a way to not fetch DisplayCurrencyQuery?
-              const resCurrency = cache.readQuery<DisplayCurrencyQuery>({
-                query: DisplayCurrencyDocument,
-              })
-              const currency = resCurrency?.me?.defaultAccount?.displayCurrency
+              const { displayCurrency, fractionDigits } =
+                getDisplayCurrencyAndFractionDigits(cache)
 
-              if (!currency) {
+              if (!displayCurrency || !fractionDigits) {
```

Comment:
FractionDigits could be 0 here so the check should `typeof fractionDigits !== "number"`